react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -85,37 +105,71 @@ class OverflowMenu extends Component<Props> {
             showLabel: true,
-            styles: this.props._bottomSheetStyles
+            styles: _bottomSheetStyles.buttons
         };
 
         return (
-            <BottomSheet onCancel = { this._onCancel }>
+            <BottomSheet
+                onCancel = { this._onCancel }
+                renderHeader = { this._renderMenuExpandToggle }>
                 <AudioRouteButton { ...buttonProps } />
                 <ToggleCameraButton { ...buttonProps } />
                 <AudioOnlyButton { ...buttonProps } />
-                <RoomLockButton { ...buttonProps } />
-                <ClosedCaptionButton { ...buttonProps } />
-                {
-                    this.props._recordingEnabled
-                        && <RecordButton { ...buttonProps } />
-                }
-                <LiveStreamButton { ...buttonProps } />
-                <TileViewButton { ...buttonProps } />
-                <InviteButton { ...buttonProps } />
-                {
-                    this.props._chatEnabled
-                        && <InfoDialogButton { ...buttonProps } />
-                }
-                <RaiseHandButton { ...buttonProps } />
-                <SharedDocumentButton { ...buttonProps } />
-                <HelpButton { ...buttonProps } />
+                <Collapsible collapsed = { !showMore }>
+                    <RoomLockButton { ...buttonProps } />
+                    <ClosedCaptionButton { ...buttonProps } />
+                    {
+                        this.props._recordingEnabled
+                            && <RecordButton { ...buttonProps } />
+                    }
+                    <LiveStreamButton { ...buttonProps } />
+                    <TileViewButton { ...buttonProps } />
+                    <InviteButton { ...buttonProps } />
+                    {
+                        this.props._chatEnabled
+                            && <InfoDialogButton { ...buttonProps } />
+                    }
+                    <RaiseHandButton { ...buttonProps } />
+                    <SharedDocumentButton { ...buttonProps } />
+                    <HelpButton { ...buttonProps } />
+                </Collapsible>
             </BottomSheet>
         );
     }
 
+    _renderMenuExpandToggle: () => React$Element<any>;
+
+    /**
+     * Function to render the menu toggle in the bottom sheet header area.
+     *
+     * @returns {React$Element}
+     */
+    _renderMenuExpandToggle() {
+        return (
+            <GestureRecognizer
+                config = {{
+                    velocityThreshold: 0.1,
+                    directionalOffsetThreshold: 30
+                }}
+                onSwipe = { this._onSwipe }
+                style = { [
+                    this.props._bottomSheetStyles.sheet,
+                    styles.expandMenuContainer
+                ] }>
+                <TouchableOpacity onPress = { this._onToggleMenu }>
+                    { /* $FlowFixMeProps */ }
```

Comment:
what is flow sad about here?

Reply:
It doesn't consider the icons as components because they are being constructed at build time by metro, do it argues that the child of the button is not a component.